### ISSUE TITLE
In text search, parents|children vanish after first leaving the buffer

### ISSUE THREAD

**Issue by JeffreyBenjaminBrown** [joshsh/brain-mode#17]
Search results have the parents and children columns displayed when they are first created, but if you navigate away from them and then return, those columns are not redrawn, only the text.

**Comment by joshsh** [comment]
Indeed.  Might as well bundle this issue with "meta columns would be better on the left-hand margin".
